daml-foundations/daml-ghc/tests/Show.daml

```diff
@@ -24,7 +24,6 @@ data Fields a = Field1 {foo : Foo, color : Optional Color} | Field2 [a]
 main = scenario do
   p <- getParty "alice"
   show p === "'alice'"
-  partyToText p === "alice"
```

Comment:
Again, I don't find this duplication unreasonable. Although this one slightly less necessary.

Reply:
I find this duplication unnecessary.